Dockerfile.centos

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2012-2016 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Dharmit Shah  - Initial implementation
+#   Mario Loriedo - Improvements
+#
+# To build it, run in the repository root:
+#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
+#
+# To run it:
+#  docker run --net=host \
+#             --name che \
+#             -v /var/run/docker.sock:/var/run/docker.sock \
+#             -v /home/user/che/lib:/home/user/che/lib-copy \
+#             -v /home/user/che/workspaces:/home/user/che/workspaces \
+#             -v /home/user/che/storage:/home/user/che/storage \
+#             registry.centos.org/eclipse/che-server
+#           
+FROM registry.centos.org/centos/centos:latest
+
+ENV LANG=C.UTF-8 \
+    JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.102-1.b14.el7_2.x86_64/jre \
```

Comment:
as you're installing "java" with yum
how can you be sure that JAVA_HOME will be installed in this folder ?
if there are updates on JAVA in centos latest image, it will break, no ?